### DISCUSSION TITLE
Set targetSOC_pct dos not work

### DISCUSSION THREAD

**Discussion by Maergsche** [tillsteinbach/WeConnect-mqtt#87] [Q&A] [answered]
Hallo, 

I try to set the "targetSOC_pct" by publishing 

`mosquitto_pub -h localhost -t weconnect/vehicles/WVWxxx/domains/charging/chargingSettings/targetSOC_pct -m 80
`
but i dosn't work.

By setting it using weconnect-cli it works: 
`weconnect-cli --username user@name.de --password pwd set /vehicles/WVWxxx/domains/charging/chargingSettings/targetSOC_pct 80`

My Topic:

`weconnect/vehicles/WVWxxx/domains/charging/chargingSettings/targetSOC_pct 50`

Does publishing the targetSOC_pct shold be workin, already?
Did I do something wrong?

PS: I am very happy about this Project ...

**Comment by diplix** [comment, marked as answer]
you'll have to use the `writetopic`:

`weconnect/vehicles/WVWxxx/domains/charging/chargingSettings/targetSOC_pct_writetopic`

**Reply by tillsteinbach** [reply]
exactly! I had to separate topics for reading/writing.
